client/src/components/AuditBoard/__snapshots__/Ballot.test.tsx.snap

```diff
@@ -144,6 +144,7 @@ exports[`Ballot renders correctly with an audited ballot 1`] = `
                           class="bp3-control bp3-checkbox sc-chPdSV gVsrQj"
                         >
                           <input
+                            checked=""
```

Comment:
Taking a sec to sanity check why this snapshot diff arose 🤔

Reply:
Okay! This makes sense. We expect this checkbox to be checked. In fact, the relevant test has explicit assertions that this is the case:

https://github.com/votingworks/arlo/blob/d154ab646120434cbb04447800c2c2884ab6218a/client/src/components/AuditBoard/Ballot.test.tsx#L55-L57

It's just that, back when we were pulling `interpretations` from state, we relied on a `useEffect` hook to properly initialize that state:

https://github.com/votingworks/arlo/blob/d154ab646120434cbb04447800c2c2884ab6218a/client/src/components/AuditBoard/Ballot.tsx#L221-L232

This meant that, on the very first render (which is what our snapshots typically consider), this checkbox wasn't checked. It was only after an (almost immediate) follow-up render due to the `useEffect` hook that the checkbox became checked.

In our new cleaner setup, the checkbox is checked on very first render :tada:

Reply:
Great investigation. That all makes sense.

In general, I'm trying to move Arlo away from snapshot tests to evaluate UI interactions (for exactly reasons like this). If you come across a snapshot, feel free to replace it with meaningful assertions and remove the snapshot.